### DISCUSSION TITLE
What happens on the Free tier after 500 tests?

### DISCUSSION THREAD

**Discussion by OzBob** [cypress-io/cypress#23130] [Questions and Help]
When a cypress project runs on the Free Dashboard tier and 

it has 501 tests. What happens on the first run?
Will the 501st result be hidden? or the 1st test result removed? 
Is there a warning or error message?

if it has 251 tests. What happens on the 2nd time the project runs the test suite?
Will all test results from the 1st run be removed?
or will the 2nd set of tests results after test #500 be hidden?

**Comment by stavares843** [comment]
hello! 👋 

based on this [issue](https://github.com/cypress-io/cypress/issues/4021#issue-436797027) seems would return an error saying

> You've exceeded the limit of private test recordings under your free plan this month. The limit is 500 private test recordings.
